samples/wasm_micro_runtime/hello_world/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.8.2)
+
+if (NOT BOARD)
+  set (BOARD qemu_x86_nommu)
+endif ()
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(wamr-hello-world)
+
+enable_language (ASM)
+
+set (WAMR_BUILD_PLATFORM "zephyr")
+
+# Build as X86_32 by default, change to "AARCH64[sub]", "ARM[sub]", "THUMB[sub]", "MIPS" or "XTENSA"
+# if we want to support arm, thumb, mips or xtensa
+if (NOT DEFINED WAMR_BUILD_TARGET)
+  set (WAMR_BUILD_TARGET "X86_32")
+endif ()
+
+if (NOT DEFINED WAMR_BUILD_INTERP)
+  # Enable Interpreter by default
+  set (WAMR_BUILD_INTERP 1)
+endif ()
+
+if (NOT DEFINED WAMR_BUILD_AOT)
+  # Enable AOT by default.
+  set (WAMR_BUILD_AOT 1)
+endif ()
+
+if (NOT DEFINED WAMR_BUILD_LIBC_BUILTIN)
+  # Enable libc builtin support by default
+  set (WAMR_BUILD_LIBC_BUILTIN 1)
+endif ()
+
+if (NOT DEFINED WAMR_BUILD_LIBC_WASI)
+  # Disable libc wasi support by default
+  set (WAMR_BUILD_LIBC_WASI 0)
+endif ()
+
+set (WAMR_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../../../modules/lib/wasm-micro-runtime)
```

Comment:
If the wasm-micro-runtime is made into a Zephyr library, that is having a: `zephyr/module.yml` file with this content:
```
name: wasm-micro-runtime

build:
  cmake-ext: True
  kconfig-ext: True
```
then we can use `ZEPHYR_WASM_MICRO_RUNTIME_MODULE_DIR` instead.

Reply:
See also: 
https://github.com/zephyrproject-rtos/loramac-node/blob/zephyr/zephyr/module.yml
https://github.com/zephyrproject-rtos/zephyr/blob/main/modules/loramac-node/CMakeLists.txt

for inspiration.

Reply:
Done.